src/Microsoft.FeatureManagement/FeatureManagementOptions.cs

```diff
@@ -13,5 +13,11 @@ public class FeatureManagementOptions
         /// If missing feature filters are not ignored an exception will be thrown when attempting to evaluate a feature that depends on a missing feature filter.
         /// </summary>
         public bool IgnoreMissingFeatureFilters { get; set; }
+
+        /// <summary>
+        /// Controls the behavior of feature evaluation when the target feature is missing.
+        /// If missing features are not ignored an exception will be thrown when attempting to evaluate them.
+        /// </summary>
```

Comment:
This comment will be published in the documentation. We should mention the default is true.

Since you are on this, can you add the default value in comments for IgnoreMissingFeatureFilters above too?

Reply:
Good idea.

Reply:
Added.